### PR TITLE
add protocol prefix to image

### LINKED ISSUE #5: Image message should add protocol prefix
Previously, in order to adapt to the http and https protocols. The url sent by the server does not have a protocol header
image

But not all image messages have no protocol header. It should be judged to add `https:` when starting with `//`. Then you can open the picture with cmd + click in vscode

### PR DESCRIPTION
**Pull request by yinxin630**
close #5 

image

### CODE REVIEW

**Comment by moonrailgun** [comment]
Hi @yinxin630 , Thanks for your pr, and i am plan to support other self-hosted fiora client. and maybe them dont support `https`.

Did you mind change your code by client serviceUrl protocol?

**Comment by yinxin630** [comment]
@moonrailgun Please take a look again

File: src/streams.ts [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+    const url = client.serviceUrl ? new URL(client.serviceUrl) : null;
+    const protocol = url?.protocol ?? 'https:';
+    return `[${msg.from.username}]: ${protocol}${msg.content}`;
+  }
+  return `[${msg.from.username}]: ${msg.content}`;
+}
+
 export function openConverseOutput(
   context: vscode.ExtensionContext,
   client: FioraClient,
```
Comment (moonrailgun) [inline src/streams.ts:19]: Hi, Here has client. Just pass it to `formatMessage` is ok

Dont try to export client to global, its should be private.

**Comment by moonrailgun** [comment]
LGTM! This will be release in v0.0.6

**Pull request merged**